app/services/random_phrase.rb

```diff
@@ -0,0 +1,19 @@
+class RandomPhrase
+  attr_reader :words
```

Comment:
not strictly no. just makes it easier to test the number of words, and for future flexibility with presentation. e.g. `phrase.words.in_groups_of(3)`